eng/scripts/GenerateReferenceAssemblies.ps1

```diff
@@ -6,5 +6,6 @@ $ErrorActionPreference = 'stop'
 $repoRoot = Resolve-Path "$PSScriptRoot/../.."
 
 & "$repoRoot\eng\common\msbuild.ps1" -ci:$ci "$repoRoot/eng/CodeGen.proj" `
+    /p:BuildNodeJs=false `
```

Comment:
Is this really OK? this is going to skip all a bunch of projects now right?

Reply:
Ah yes, this would change the dev behaviour. The skipping of dependent projects is only done when -ci is not specified. I tried it locally and it seems like specifying -ci should be safe. Any concerns with that approach?

Reply:
Actually, I don't think this is needed anymore.

Reply:
That doesn't sound OK.

We want things to be simple to use - what's the actual problem that needs to be solved?

Reply:
I meant update the scrip to pass -ci (instead of -ci:$ci) by default in this script. Nothing changes on the user's end. 

In any case, I ended deciding that it shouldn't be necessary to skip building NodeJS. Or at least, I need to be reminded why it was needed. What's likely to happen is that I just need to install NodeJS on the CI build.

Reply:
`-ci` is intended to inform the scripts that they're running in AzDO, not for workarounds when building locally. The scripts should _all_ work with and without `-ci`.

Reply:
Ah I remember, there's a target that gets invoked specifically for generating reference assemblies. The target is defined for c sharp projects but not in NPM projects. I disabled building NodeJS because I don't think generating reference assemblies is relevant. As for not skipping csprojs, if we can't use the -ci trick. I'll need to either 1. define a dummy target in NPM.targets, or 2. add a specific option to opt out of building dependent npm projects when generating reference assemblies. Both feel a little out of place but I prefer 1.

Reply:
Actually it works now, so I'm not sure what I was running into before. Oh well.